Add Japanese translations

tomodachi94 opened the pull request:
Not sure how many Japanese players you have, but this could be useful if you ever have some.

tomodachi94 committed:
l10n_fuel: Add Japanese translation

tomodachi94 committed:
l10n_torch: Add Japanese translation

tomodachi94 committed:
l10n_strip: Add Japanese translation

tomodachi94 committed:
l10n_reactor: Add Japanese translation

Chi-Yu commented:
We have zero players at this point.

Does ComputerCraft even exist anymore? I think the last release was 5 years ago.

Chi-Yu commented on src/api/l10n-translations/ja/l10n_reactor:
```diff
@@ -0,0 +1,37 @@
+t_messagesOnline = '発電する...'
+t_messagesOffline = '待機...'
+t_messagesNoFuel = '燃料ではない'
+t_messagesBufferFull = 'Buffer full'
```
Is there a specific reason why you didn't translate these 4 strings?

Chi-Yu commented on src/api/l10n-translations/ja/l10n_reactor:
```diff
@@ -0,0 +1,37 @@
+t_messagesOnline = '発電する...'
+t_messagesOffline = '待機...'
+t_messagesNoFuel = '燃料ではない'
+t_messagesBufferFull = 'Buffer full'
+t_startButtonTitle = 'Status Control:'
+t_startButtonStart = ' start '
+t_startButtonStop = ' stop  '
```
Is there a specific reason why you didn't translate these 4 strings?

tomodachi94 replied:
Whoops, I missed those. My bad!

tomodachi94 commented:
> Does ComputerCraft even exist anymore? I think the last release was 5 years ago.

ComputerCraft (the original project) is mostly dead; however, some other people made a fork called [CC: Tweaked](https://github.com/cc-tweaked/CC-Tweaked), which is much more active.

Chi-Yu commented:
I cross checked your translations with Google Translate and it seems Google's grammar translation is horrible. I trust you that it doesn't sound as weird to someone who actually speaks Japanese. 😉

Add the missing ones and I can merge this, 7 years later.

I'm curious. Do you actually use these scripts somewhere?

tomodachi94 commented:
> I cross checked your translations with Google Translate and it seems Google's grammar translation is horrible. I trust you that it doesn't sound as weird to someone who actually speaks Japanese. wink

If you feel a pressing need, you can paste in the sentances to [Jisho.org](https://jisho.org) and verify that they are correct.

> I'm curious. Do you actually use these scripts somewhere?

I use the `strip` script occasionally when playing.

tomodachi94 committed:
l10n_reactor: Add missing Japanese translations

Here's a list of what was missing:
- t_messagesBufferFull
- t_startButtonTitle
- t_startButtonStart
- t_startButtonStop

tomodachi94 commented:
@Chi-Yu I fixed the missing translations.

Chi-Yu commented:
I was hoping that someone actually used the Big Reactor script, but I guess there are way more sophisticated ones out there which do a better job.

Thanks for the contribution though.

Merged.
